### ISSUE TITLE
Add new slot as container of options

### ISSUE THREAD

**Issue by matjaz321** [sagalbot/vue-select#887]
**Is your feature request related to a problem? Please describe.**
I need to implement Vue perfect scrollbar in the dropdown of vue-select, but it currently doesn't support slot which would behave as container of all slots.

**Describe the solution you'd like**
Add new slot as container all options

**Comment by seyfer** [comment]
Yes, currently it is not possible to wrap the list of options into any other container via slot. But we need it for use of vue-perfect-scrollbar, for example.

So we need this code
`<ul ref="dropdownMenu" `
transformed into slot or wrapped into it

**Comment by doutatsu** [comment]
@sagalbot Any chance this will get released anytime soon? Everything about the library has been great, but this is the last crucial bit that I need to fully migrate

**Comment by doutatsu** [comment]
Bumping this again

**Comment by romeomihalovics** [comment]
You can find a workaround for this issue [here](https://github.com/sagalbot/vue-select/issues/1133#issuecomment-1374027456)
